### ISSUE TITLE
Symbols missing in symbols viewer (python example)

### ISSUE THREAD

**Issue by suchabody** [geany/geany#2999]
Symbols viewer incomplete with the following Python code.
```
class Foo:
    Bar = 'Xyzzy'
    Baz = 'Offending character >>> = <<<'
```
No symbol Baz but remove that final '=' and it appears.

**Comment by elextr** [comment]
There will be a new version of the ctags parser merged to git "soon"(ish), if that doesn't fix it the problem should be reported upstream.

**Comment by eht16** [comment]
This will be fixed by #3031.

**Comment by techee** [comment]
Fixed by #3031, closing.
